translations/fr.po

```diff
@@ -93,258 +89,166 @@ msgstr ""
 msgid "Source code and document"
 msgstr "Code source et document"
 
-#: config.py:207 config.py:224 config.py:279
+#: config.py:207 config.py:224 config.py:275
 msgid "Saving customized lemmas"
 msgstr "Sauvant les lemmes personalisés"
 
-#: config.py:243
```

Comment:
Line 100 to 190 need to be preserved for the EPUB Word Wise language drop down button, these texts are not hard coded in the Python file so Poedit always moves them commented to the end when clicking the "Update from source code" button.

Reply:
Oops thought Poedit updating from source was safe. Lets uncomment these then.